bin/sl-meshctl.js

```diff
@@ -144,6 +144,10 @@ function cmdStatus(instance) {
     fmt(1, 'port', '%s', rsp.port);
     fmt(1, 'base', '%s', rsp.base);
 
+    if (rsp.version) {
```

Comment:
Should this check be more elaborate? The format of `rsp.version` theoretically depends on the value in `rsp.version.api`...